uint32 steps

---

uint32 old_iterations
uint32 new_iterations
